src/shipit_uplift/shipit_uplift/coverage.py

```diff
@@ -111,7 +111,8 @@ def _get(url=''):
 
     @staticmethod
     async def get_file_coverage(changeset, filename):
```

Comment:
The changes should probably be done in src/shipit_uplift/shipit_uplift/coverage_for_file_impl.py, where you have the latest build from `get_coverage_build` (look at coverage_by_changeset_impl.py, which is doing the same).

Reply:
ok, I did it.
I changed hg_changeset to build_changeset and git_changeset to git_build_changeset to have same names as in coverage_by_changeset.